semgrep/semgrep/commands/install.py

```diff
@@ -0,0 +1,105 @@
+import shutil
+import sys
+from pathlib import Path
+
+import click
+import requests
+from tqdm import tqdm
+
+from semgrep.commands.login import Authentication
+from semgrep.constants import SEMGREP_URL
+from semgrep.constants import SEMGREP_USER_AGENT
+from semgrep.error import FATAL_EXIT_CODE
+from semgrep.error import INVALID_API_KEY_EXIT_CODE
+from semgrep.semgrep_core import SemgrepCore
+from semgrep.util import set_flags
+from semgrep.verbose_logging import getLogger
+
+logger = getLogger(__name__)
+
+
+@click.group()
+def install() -> None:
```

Comment:
rename to `deep-semgrep-install` or something?

Reply:
Renamed to install-deep-semgrep